statsmodels/genmod/families/family.py

```diff
@@ -1432,3 +1432,193 @@ def resid_anscombe(self, endog, mu):
         return ((hyp2f1(-self.alpha * endog) - hyp2f1(-self.alpha * mu) +
                  1.5 * ( endog**(2 / 3.) - mu**(2 / 3.))) /
                 (mu + self.alpha * mu**2)**(1 / 6.))
+
+
+class Tweedie(Family):
+    """
+    Tweedie family.
+
+    Parameters
+    ----------
+    link : a link instance, optional
+        The default link for the Tweedie family is the log link when the
+        link_power is 0. Otherwise, the power link is default.
+        Available links are log, cloglog, identity, nbinom and power.
+        See statsmodels.family.links for more information.
+    var_power : float, optional
+        The variance power.
+    link_power : float, optional
```

Comment:
Sounds good. You might also want to look at [this line](https://github.com/statsmodels/statsmodels/blob/master/statsmodels/genmod/families/family.py#L1254) of code. I think we really might want `if link is L.NegativeBinomial:` I don't think that this works as currently written. (Mind you, I don't use Negative Binomial, so I'm not 100% sure here.)